core/src/main/java/org/apache/accumulo/core/logging/FateLogger.java

```diff
@@ -175,6 +176,21 @@ public Optional<FateTxStore<T>> createAndReserve(FateKey fateKey) {
         }
         return txStore;
       }
+
+      @Override
+      public boolean isReserved(FateId fateId) {
+        return store.isReserved(fateId);
+      }
+
+      @Override
+      public Map<FateId,FateReservation> getActiveReservations() {
+        return store.getActiveReservations();
+      }
+
+      @Override
+      public void deleteDeadReservations() {
+        store.deleteDeadReservations();
```

Comment:
Probably would not go here, but it would be nice to have consistent logging somehow for each dead reservation that is detected and deleted.

Reply:
Not sure if this is workable,  maybe the method could take a consumer that is passes found dead reservations to  like : `deleteDeadReservations(Consumer<FateReservation> foundDead)` then this could add logging for those.  Alternatively could just put the same log stmts in each impl of the store.

Reply:
Yes, nice suggestion. For simplicity in this new commit, I just added the same log message for both impls.